skyxplore-ui-test/src/test/java/com/mentoring/pageobject/EquipmentPage.java

```diff
@@ -61,7 +60,7 @@ public void moveStorageCex01ItemToShipEmptySlot() {
     }
 
     @Step("Getting the number of extra free slots from CEX-01 item description")
-    public int getCex01ItemExtraFreeSlots() throws ExtraFreeSlotsNumberNotFoundException {
+    public int getCex01ItemExtraFreeSlots() throws AssertionError {
```

Comment:
This is an assertion error, not an unhandled exception. You only need to add "throws" to your method when it's an unhandled exception (or you can try-catch it).

Reply:
Ok, fixed, thx.